main.py

```diff
@@ -1,13 +1,14 @@
-from logic import *
-
-#Start your program here
-
-a = 1
-
-def b():
-    global a
-    a += 2
-    print( " " + str(a))
-
-b()
-print(a)
\ No newline at end of file
+sum_list = []
+
+while True:
+    number_1 = int(input ("What is number 1"))
```

Comment:
Don't leave a space after `input`. 

Try to get rid of all the yellow highlight in the program.

Reply:
I recently fixed this but I haven't pushed it yet expect that push before next Sunday